File: sdkwork-app-sdk-csharp/Models/CartStatisticsVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class CartStatisticsVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? CartId { get; set; }
        public int? ItemCount { get; set; }
        public int? TotalQuantity { get; set; }
        public double? TotalPrice { get; set; }
        public int? SelectedItemCount { get; set; }
        public int? SelectedQuantity { get; set; }
        public double? SelectedPrice { get; set; }
    }
}

File: sdkwork-app-sdk-typescript/src/types/cart-statistics-vo.ts
/** 购物车统计响应 */
export interface CartStatisticsVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 购物车ID */
  cartId?: string;
  /** 商品种类数 */
  itemCount?: number;
  /** 商品总数量 */
  totalQuantity?: number;
  /** 商品总价格 */
  totalPrice?: number;
  /** 选中商品种类数 */
  selectedItemCount?: number;
  /** 选中商品总数量 */
  selectedQuantity?: number;
  /** 选中商品总价格 */
  selectedPrice?: number;
}
